webpack.config.js

```diff
@@ -10,21 +10,33 @@ const {version} = require('./packages/node_modules/webex/package.json');
 dotenv.config();
 dotenv.config({path: '.env.default'});
 
-module.exports = (env = process.env.NODE_ENV || 'production', args) => ({
-  entry: env === 'development' ?
-    `${path.resolve(__dirname)}/packages/node_modules/webex/src/index.js` :
-    './packages/node_modules/webex',
-  mode: env === 'development' ? 'development' : 'production',
+/**
+ * Webpack Config
+ *
+ * @param {object} [env]
+ * @param {string} env.NODE_ENV
+ * @param {*} [args]
+ * @returns {object}
+ */
+module.exports = (env = {NODE_ENV: process.env.NODE_ENV || 'production'}, args) => ({
```

Comment:
Wouldn't `env` come from the `args` now that you're passing it as an argument to the CLI?

Reply:
This is more for `wdio.config.js` usage where it's imported as a config